pytest_timeouts.py

```diff
@@ -113,13 +113,23 @@ def pytest_runtest_call(self, item):
 
     @staticmethod
     def fetch_marker_timeout(item, name):
-        marker = item.get_closest_marker(name=name)
-        if marker is None:
-            return None
-        if not marker.args:
-            raise TypeError('Timeout value is missing')
-        timeout = float(marker.args[0])
```

Comment:
Convert to float is redundant in that place. `parse_timeout` do it for every source of timeout value.